lib/reporters/mini.js

```diff
@@ -215,7 +215,7 @@ MiniReporter.prototype.finish = function (runStatus) {
 		});
 	}
 
-	if (runStatus.failFastEnabled === true) {
+	if (runStatus.failFastEnabled === true && runStatus.remainingCount > 0) {
```

Comment:
This should check if there were any failures. Otherwise the warning is also printed when `.only()` is used and `--fail-fast` is on:

```
$ "$(npm bin)"/ava --fail-fast

  1 passed

  `--fail-fast` is on. Any number of tests may have been skipped

  The .only() modifier is used in some tests. 135 tests were not run.
```